target/linux/ath79/dts/qca9533_tplink_tl-wr842n-v3.dts

```diff
@@ -24,6 +24,7 @@
 			label = "tp-link:green:power";
 			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
 			default-state = "on";
+			linux,default-trigger = "heartbeat";
```

Comment:
This should be removed, also from 1043

Reply:
@obsy As using this feature should not be the default? Or is there any other reason? 
I personally like this feature.

Reply:
Power led should be displayed on. If somebody want use heartbeat - it's easy to add by yourself in userland. We should not impose a led flashing. And openwrt does not use it by default, it tries to keep the leds working as in the original software.